### ISSUE TITLE
Cannot find parent element

### ISSUE THREAD

**Issue by clieee** [craftcms/feed-me#607]
### Description
When importing from Google Sheet it has problem finding the parent element (for a structure). 
I'm using three (3) different import feeds. Google sheet is structured like this:
Section,Category,Question
section-title-1,category-title-1,question-title-1

1) Import root elements, called "Section"
2) Import categories with parent mapped to title of section
3) Import questions with parent mapped to title of category

### Steps to reproduce

a) Import the sheet as it is, will get error: ```There was a problem getting the parent element. - Structures.php: 238``` (title for section in this case is "Legal"). Just a few  elements will get imported. 

b) Import sheet with different section title and it works

I havent had any issues with a different sheet, feels hard to debug. Any ideas?

### Additional info

- Plugin version: 4.1.2
- Craft version: 3.3.4.1

**Comment by clieee** [comment]
This was solved by removing the section completely from Craft, and then create a new section (same structure etc). I guess something went wrong from deleting, adding etc entries that caused the import error.

**Comment by Friksel** [comment]
Not sure if it is the same issue as I am facing right now, but here I get the exact same error message and the parentItem does exist, as it is just imported by the same Feedme feed in front of the children.

Just 5 items;
- first item has a CustomID which is a unique ID from the CMS the data was coming from and bound to a custom field in the entry by feedme
- All next items in the json feed should be children of the first item and therefore have a field called 'ParentCustomID' which is the CustomID of the first item. In feedme this ParentCustomID is bound to the ParentID with the dropdown set to the CustomID field to find a match.

This only imports the first item, where ParentCustomID is just null and skips the rest.

Checked everything and I'm sure this should work. For other entries I also used the ParentID, but there I synchronised it on a Title-field, so not on a custom ID. 

Also it's a multilanguage site, so the same CustomID is already used a lot to sync up different languages and works fine with that.

Seems like there is some issue here
